proof-systems/src/darlin/marlin/src/iop/mod.rs

```diff
@@ -109,7 +110,7 @@ impl<F: PrimeField> IOP<F> {
     /// via an `EvaluationProvider` (as defined below).
     // Note: To complete Marlin verification, one still has to check the opening proofs.
     #[allow(non_snake_case)]
-    pub(crate) fn verify_sumchecks(
+    pub fn verify_sumchecks(
```

Comment:
A small note here: for the recursion related Marlin variants we need to drop the optimization of not comming the T poly thus we need again, explicitly, both sumchecks.  Here you don't do any inner sumcheck aggregation so, conceptually, it's still fine. For sake of consistency and for those who are gonna review this, I would suggest to leave a comment explicitly saying that, as this is a temporary increment in the software, we decide not wasting time in restoring both sumchecks but, as soon as we introduce inner sumcheck aggregation, that must be done.